pricegraph/src/orderbook.rs

```diff
@@ -233,12 +298,13 @@ impl Orderbook {
                 )
             });
 
-        Some(capacity)
+        Some((capacity, price))
     }
 
-    /// Gets the maximum capacity of a path expressed in an amount of the
-    /// starting token. Returns `None` if the path doesn't exist.
-    fn find_maximum_capacity(&self, path: &[NodeIndex]) -> Option<f64> {
+    /// Finds a transient trade along a path and returns the maximum capacity of
+    /// the path expressed in an amount of the starting token and the transient
+    /// price. Returns `None` if the path doesn't exist.
+    fn find_transient_trade(&self, path: &[NodeIndex]) -> Option<(f64, f64)> {
```

Comment:
naming nit: What about this trade is transient? Maybe calling it `find_maximum_capacity_and_price` or `find_best_trade`